package.json

```diff
@@ -23,7 +23,7 @@
     "nested-error-stacks": "^2.0.0",
     "node-zookeeper-client": "~0.2.2",
     "optional": "^0.1.3",
-    "retry": "~0.6.1",
+    "retry": "~0.7.0",
```

Comment:
Absolutely. I just bumped to 0.7.0 trying to be conservative. But I have just checked the latest version, `0.10.0` and has no breaking changes.
So updating my fork changing the commit that has that bump on the retry dependency.